corehq/apps/smsforms/management/commands/migrate_sms_sessions_to_sql.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand
+from corehq.apps.smsforms.models import XFormsSession, sync_sql_session_from_couch_session, SQLXFormsSession
+from dimagi.utils.couch.database import iter_docs
+
+
+class Command(BaseCommand):
+    args = ""
+    help = ""
+
+    def handle(self, *args, **options):
+        db = XFormsSession.get_db()
+        session_ids = [row['id'] for row in db.view("smsforms/sessions_by_touchforms_id")]
```

Comment:
I haven't done much querying of just ids so I don't know if this will be an issue or not, but do you think this query will be a problem if there are 375k sessions?